docs/history-framework.md

```diff
@@ -193,7 +193,113 @@ sequenceDiagram
   deactivate TreeManager
 ```
 
-## TODO:
+## Serialization
+We store the history of changes to a document in Firestore. Each history entry is stored in a separate Firestore document. These history entry documents are stored in the same way that comments are stored. There is a parent Firestore document that has meta data about the actual CLUE document and then under this parent Firestore document is a collection for the comments and a collection for the history entries.
+
+The history entries are sent up by the TreeManager which is an MST model. In other places we interact with Firestore through react components so we can use hooks. Since we are in the model we just work directly with Firestore.
```

Comment:
```suggestion
The history entries are written by the TreeManager which is an MST model. In other places we interact with Firestore through react components so we can use hooks. Since we are in the model we just work directly with Firestore.
```